ridam.txt

```diff
@@ -0,0 +1 @@
+I am new here
```

Comment:
Please read the `Contributing` heading of our [readme](https://github.com/do-community/cloud_haiku/blob/master/README.md) to submit to this repo.